### PR TITLE
mattdrayer/add-author: Add "mattdrayer" as author

### PR DESCRIPTION
**Pull request by mattdrayer**
**Description:** Adding myself to the AUTHORS file -- can't believe I haven't done it yet!

**JIRA:** N/A

**Dependencies:** N/A

**Merge deadline:** SOON

**Merge checklist:**

~~- [ ] Regenerate requirements with `make upgrade && make requirements` (and make sure to fix any errors).~~
- [ ] All reviewers approved
- [x] CI build is green
~~- [ ] Version bumped~~ (will do as part of #99)
~~- [ ] Changelog record added~~
~~- [ ] Documentation updated (not only docstrings)~~
- [x] Commits are (reasonably) squashed
~~- [ ] Translations are updated~~
- [x] PR author is listed in AUTHORS

**Post merge:**
~~- [ ] Create a tag~~
~~- [ ] Check new version is pushed to PyPi after tag-triggered build is finished.~~
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** Only that I'll be doing less authoring going forward  😄

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -9,3 +9,4 @@ Grant Goodman <ggoodman@edx.org>
 Zubair Afzal <zubair.afzal@arbisoft.com>
 Douglas Cerna <douglas@opencraft.com>
 Tim Krones <tim@opencraft.com>
+Matt Drayer <mattdrayer@edx.org>
```
Comment (haikuginger) [inline AUTHORS]: Newline, please.

Comment (mattdrayer) [inline AUTHORS]: Added!

**Comment by mattdrayer** [comment]
Hah!  😃

**Comment by haikuginger** [comment]
💯

**Pull request merged**